### PR TITLE
fix: show first icon in opportunity cards

### PR DESCRIPTION
**Pull request by NeOMakinG**
## Description

The opportunity cards containing more than one icons are not looking so good right now, let's show the first icons so it's sexier!

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

no sisue

Low risk, presentaitonal

> What protocols, transaction types or contract interactions might be affected by this PR?

## Testing
Go to the earn page, look at the ETH/FOX opportunity, it should be displayed properly

### Engineering
n/a

### Operations

n/a

## Screenshots (if applicable)
Before:
image
image

After:
image
image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Discussed with @NeOMakinG and she is happy and ready for review pending release rebase

File: src/components/StakingVaults/Cells.tsx [hunk 1]
```diff
@@ -99,9 +101,9 @@ export const AssetCell = ({
       <HStack flex={1}>
         <SkeletonCircle isLoaded={!!asset} mr={2} width='auto' height='auto'>
           {icons && icons.length > 1 ? (
-            <PairIcons icons={icons} iconSize='sm' bg='none' />
+            <PairIcons icons={icons} iconSize='sm' bg='none' showFirst={true} />
```
Comment (gomesalexandre) [inline src/components/StakingVaults/Cells.tsx:104]: super nitpick but possibly spread `pairProps` here for reuse, though happy with the declarativeness of this as well

File: src/components/StakingVaults/Cells.tsx [hunk 2]
```diff
@@ -99,9 +101,9 @@ export const AssetCell = ({
...
           ) : (
-            <AssetIcon assetId={asset.assetId} size='md' />
+            <AssetIcon assetId={asset.assetId} size='md' pairProps={pairProps} />
```
Comment (gomesalexandre) [inline src/components/StakingVaults/Cells.tsx:106]: Consumed in quite a few places, test me against release there
image

Comment (gomesalexandre) [reply src/components/StakingVaults/Cells.tsx:106]: ✅ 
While it *is* consumed in a bunch of places (fiat ramps, markets page, your loans, available pools, related assets...) you won't see LP assets there, so effectively a low surface area - gave it some smoke test in all those places and couldn't get to any LP assets even after going many, many pages on fiat ramps/markets

**Review by gomesalexandre — APPROVED** [review]
Tested against release (right), LGTM! Get in ser.

- Opportunity cards look better with the two assets shown 🎉 

image

- Opportunity rows also look better

image
image

- Portfolio looks much better with multi-asset icons *or* single/pair icons

image

File: src/components/StakingVaults/Cells.tsx (hunk 1 above)
Comment (NeOMakinG) [reply src/components/StakingVaults/Cells.tsx]: Done in [a3ef322](https://github.com/shapeshift/web/pull/7488/commits/a3ef322255b2188c79f85b92961b44cf7c5a651e)

**Pull request merged**